website/client/src/components/groups/questSidebarSection.vue

```diff
@@ -248,35 +303,88 @@
 
   .no-quest-section {
     padding: 2em;
-    color: $gray-300;
 
     h4 {
-      color: $gray-300;
+      margin-bottom: 0;
     }
 
     p {
-      margin-bottom: 2em;
+      margin-bottom: 1em;
+      color: $gray-100;
+      font-size: 0.75rem;
+      line-height: 1.33;
     }
 
-    .svg-icon {
-      height: 30px;
-      width: 30px;
-      margin: 0 auto;
-      margin-bottom: 2em;
+    .quest-icon {
+      width: 1.125rem;
+      height: 1.25rem;
+      margin: 0 auto 0.5em;
+      object-fit: contain;
+      border-radius: 2px;
+      color: $gray-200;
     }
   }
 
+  .quest-pending-section {
+    display: flex;
+    margin-bottom: 0.5rem;
+
+    .titles {
+      flex: 1;
+      margin-top: 1rem;
+      font-size: 0.75rem;
+      line-height: 1.33;
+
+      strong {
+        display: block;
+        min-height: 1rem;
+        font-weight: bold;
+        font-size: 0.75rem;
+        line-height: 1.33;
+        color: $gray-100;
+        margin-bottom: 0.25rem;
+      }
+
+      .members-invited {
+        min-height: 1rem;
+        color: $blue-10;
+        margin: 0;
+
+        &:hover, &:focus {
+          color: $blue-10;
+          text-decoration: underline;
+        }
+      }
+    }
+
+    .quest-icon {
+      width: 4.25rem;
+      height: 4.25rem;
+    }
+  }
+
+  .full-width {
+    width: 100%;
```

Comment:
Doesn't Bootstrap's `w-100` already do this?

Reply:
can try it - didn't know about the `w-100` 

for some of the bootstraps classes I tend not to use them - some just can't be bend to make the design perfect :D 


but the `w-100` will probably work then - gonna check if there are other places with that as well